### ISSUE TITLE
Could someone explain how to install?

### ISSUE THREAD

**Issue by enrique131** [McCloudS/subgen#4]
Hi,
I am able to follow some of the instructions:
Plex and tautulli are connected (both in docker in synology). I can create the webhook (i have another to send to trakt.tv what i watch on plex working) with no issues. 
But the rest i am not sure how to proceed, when i read this: 
"**The docker-compose/Dockerfile settings are relatively straightforward (and poorly commented).**" i am lost. 

I am thinking maybe without the docker, via terminal adding:
"wget https://raw.githubusercontent.com/McCloudS/subgen/main/subgen/subgen_nodocker.py
apt-get update && apt-get install -y ffmpeg git gcc python3
pip3 install webhook_listener
python3 -u subgen_nodocker.py"

but then how can i edit the volumes locations? 
Sorry if that is too obvious. If someone can point me in the right direction i will appreciate it.

**Comment by McCloudS** [comment]
Are you using docker-compose or synology's built in webui?

**Comment by enrique131** [comment]
Synology webui is a bit more fail-safe for me. So i think i managed to install it. But even with the same volume as Plex (only needed the shows) it breaks my Plex. I am unable to play anything and it break my menu. If i disable McCloudS/subgen, it works again. Not sure what i am doing wrong

**Comment by McCloudS** [comment]
I have some pretty substantial upgrades coming down the road, like removing Tautulli and solely using Plex webhooks.  I'll let you know.  It should be easier to install as well.

**Comment by McCloudS** [comment]
Give it another shot using the docker image in the instructions.  The only thing that MUST be set are going to be network mode in host, and the PLEXTOKEN/PLEXSERVER environment values.  I assume you should be able all of that in DSM, but i'm not familiar with it.

**Comment by enrique131** [comment]
You kill me here... webhooks are for plex pass and i wont get plex pass until they allowed me to skip less than 30 seconds... is a personal issue for me. 
I managed to follow the new instructions and i can tell you that i can install, and plex works with the docker is active, but it fails too.   
Ram is at 860Mb but i don't know if i have to wait to it to sub the whole tvshow folder or how it works now. 
Basically it slows down the NAS so much is unusable. 
I am sure i am doing 100 things wrong.

**Comment by McCloudS** [comment]
Sounds like you're RAM and CPU constrained.  The whisper model heavily uses both.

**Comment by enrique131** [comment]
I am using this: https://www.synology.com/en-us/products/DS220+#specs
So maybe you are right. Thanks for your time.
